Tests/FilterTest.cs

```diff
@@ -171,10 +179,15 @@ async Task SendTo(string state)
     // For the producer side we have the ConfirmationHandler the messages with errors 
     // will be reported as not confirmed and the user can handle them.
     // for the consumer the messages will be skipped and logged with the standard logger
-    [Fact]
+    [SkippableFact]
```

Comment:
use `[Fact(Skip="reason")]`

Reply:
I disagree. We only want to skip this test if the broker is < 3.13. We want to run always run if the broker is >= 3.13

Reply:
ops, yes @Zerpet you are right